src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+
+### 2\. Create the interface{#create-UI}
+Now create the UI to interact with unity
+  1. Create a ```Canvas```.
+  2. Create a ```Text``` with name **BalanceText** to store the balance.
+  3. Create a ```Button```to call method to send a transaction and a ```Input Field``` to store the address for transaction.
+![Captura de Tela (90)](https://user-images.githubusercontent.com/52639395/119541875-4dd3ba00-bd65-11eb-913a-ba320f0dfc55.png)
+
+### 3\. Write code to receive the balance {#unity-balance}
+In asset folder create a new cs file ```BalanceController.cs```, drag and drop this file into gameobject ```BalanceText```,open the code file and writer the code below.
+```csharp
+    void EtherBalanceToUnity(string amount)
+    {
+        GameObject.Find("BalanceText").GetComponent<Text>().text = $"Balance: {amoun} EHT";
+    }
+```
+The **EtherBalanceToUnity** will receive the account balance from react app.
+
+### 4\. Write code to send a transaction {#unity-send-transaction}
+In asset folder create a new folder with name **Plugins**, create a new file ```script.jslib``` inside this new folder, 
+[jslib](https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html) script are similar  to javascript,
+this script build a bridge between your Unity project and the react app,**SendTranscationFromUnitya** is the function that will exist in both unity and react app,write the code below.
+```javascript
+    mergeInto(LibraryManager.library, {
+      SendTranscationFromUnity:function (address,amount) {
+        ReactUnityWebGL.SendTranscationFromUnity(Pointer_stringify(address), amount);
+      },
+});
+```
+Create a new file ```SendController.cs``` put the code in the gameobject **Button**,open the file and add this code.
+```csharp
+    //The variable that stores the input field.
+    public InputField input;
+    //Tell to unity to use the script.jslib in Plugins folder, to use the SendTranscationFromUnity function in script.jslib . 
+    [DllImport("__Internal")]
+    private static extern void SendTranscationFromUnity(string address, double amount);
+    void Start()
+    {
+        //Add a method to send information about your transaction.
+        GameObject.Find("Button").GetComponent<Button>().onClick.AddListener(Transaction);
+        //Set the input field.
+        input = GameObject.Find("InputField").GetComponent<InputField>();
+        //Since unity Webgl does not support ctrl-v without external code, set demo address here.
+        input.text = "0xf8331f18a7106bF6B9C0847b3BbC5B6180806A2C";
+    }
+    void Transaction()
+    {
+        //A valid eth address is 42 characters long 
+        if (input.text.Length == 42)
+        {
+          //Send transaction information to react app.
+          SendTranscationFromUnity(input.text, 1e15);
+        }
+    }
+```
+
+Now our unit project is ready, wait for the next tutorial part to build the project, the code only works when is builded, it will not work in unity editor mode, 
```

Comment:
Consider something like:
"Now our Unity project is ready. In the next step we will build the project to allow us to interact with the project."